keyboards/boardsource/equals/equals.c

```diff
@@ -0,0 +1,19 @@
+// Copyright 2023 @boardsource
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+extern void ui_init(void);
+extern void ui_task(void);
```

Comment:
Not sure this is the best way to handle this.  

But extern isn't needed here: 

```suggestion
void ui_init(void);
void ui_task(void);
```